contractcourt/htlc_incoming_contest_resolver.go

```diff
@@ -166,6 +167,26 @@ func (h *htlcIncomingContestResolver) Resolve() (ContractResolver, error) {
 	}
 }
 
+func (h *htlcIncomingContestResolver) report() *ContractReport {
+	// No locking needed as these values are read-only.
+
+	var finalAmt = h.htlcAmt.ToSatoshis()
+	if h.htlcResolution.SignedSuccessTx != nil {
+		finalAmt = btcutil.Amount(
+			h.htlcResolution.SignedSuccessTx.TxOut[0].Value,
+		)
+	}
+
+	return &ContractReport{
+		Outpoint:       h.htlcResolution.ClaimOutpoint,
+		Incoming:       true,
+		Amount:         finalAmt,
+		MaturityHeight: h.htlcExpiry,
```

Comment:
Perhaps we also want to show the number of blocks until it's mature? Can likely compute that at the RPC level before passing it onto the caller.

Reply:
It is currently calculated inside `rpcserver`.

Reply:
Yep, missed that!